docs/install.md

```diff
@@ -39,16 +39,38 @@ you'll need to flip the `extensions.legacy.enabled` preference, too.
 To **run the extension in a Firefox Nightly** browser, you can run:
 
 ```sh
-npm run-script run -- -b nightly
+npm run run -- -b nightly
 ```
 
+This will automatically create a fresh new user profile that will not persist
+between runs. This means the data will be lost every time.
+
+## Running the extension with a persistent profile
+
+To **run the extension with a profile that persists** between runs, you can
+create a new profile by browsing to `about:profiles`.
+
+Once you have a new profile created (no matter the location), you can tell jpm
+(via npm) to run using that profile _and_ not to copy the profile temporarily
+so that any changes (e.g. adding new entries) will be saved:
+
+```sh
+npm run run -- -p /path/to/profile --no-copy
```

Comment:
`-p profile-name` is just as acceptable (and arguably easier to remembertype) as  `/path/to/profile`.  The latter is necessary if the profile isn't somewhere Firefox already knows to find it